conf.d/git.fish

```diff
@@ -3,6 +3,11 @@
 functions -e _git_install _git_update _git_uninstall
 
 # fisher initialization, protected as omf also tries to run it.
+set -l __fish_config_dir $__fish_config_dir
+if set -q fisher_path
+  set __fish_config_dir $fisher_path
+end
+
 if test -f $__fish_config_dir/functions/__git.init.fish
   source $__fish_config_dir/functions/__git.init.fish
   __git.init
```

Comment:
I didn't know about this fisher_path, how about we use it instead for the rest of the file, something like (untested) would be easier to follow:

```suggestion
set -q fisher_path; or set -l fisher_path $__fish_config_dir
if test -f $fisher_path/functions/__git.init.fish
  source $fisher_path/functions/__git.init.fish
  __git.init
```

Reply:
This works fine, I've pushed it